benchmarks/distributed/lambda_runtimes/cpp_runtime/update.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+NAME="test-python"
+BUILD_DIR=../../../build-py38-haswell
```

Comment:
this looks like a magic folder - sure this works for anyone trying to replicate your setup?